app/src/main/java/org/fossasia/openevent/general/event/EventDetailsViewModel.kt

```diff
@@ -27,6 +33,14 @@ class EventDetailsViewModel(private val eventService: EventService, private val
     private val mutableEvent = MutableLiveData<Event>()
     val event: LiveData<Event> = mutableEvent
 
+    fun fetchEventSpeakers(id: Long) {
+        speakerService.fetchSpeakersForEvent(id)
+    }
+
+    fun loadEventSpeakers(id: Long): LiveData<List<Speaker>> {
+        return speakerService.fetchSpeakersFromDb(id)
```

Comment:
Store this in a variable after the first time and return the same instance

Reply:
like this?
  val speakerList = speakerService.fetchSpeakersFromDb(id)
        return speakerList